### PR TITLE
[DDW-589] Fix native tokens UI/UX issues

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR fixed text overlapping issue on native token input field.
[Jira Ticket](https://jira.iohk.io/browse/DDW-589)

## Todos
- [x] Update "Minimum ada tooltip message" when there are no native tokens added [@darko-mijic]
- [x] Apply the same dynamic width calculation for the ada amount field too [@yakovkaravelov]
- [x] Fix text overlapping issue on native token input field.
- [x] Fix PopOver properties (`isShowingOnHover={false}` should be removed and `interactive={true}` added) [@aleksandardjordjeviciohk]
image-2
image
image-1

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1614681327128000)

### App
- [x] Go to `Wallet` => `Send`, click on `Add token` button to generate token input field and make sure token input field text is not overlapped with addon button text when filling in big value

### English
image

### Japanese
image

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Great work @yakovkaravelov . Tested on 16889

**Review by ManusMcCole — APPROVED** [review]

File: source/renderer/app/components/wallet/WalletSendForm.js [hunk 1]
```diff
@@ -804,167 +800,27 @@ export default class WalletSendForm extends Component<Props, State> {
...
-                              getStakePoolById={() => {}}
-                              errorPosition="bottom"
-                            />
-                          </div>
-                        </div>
-                      </div>
-                    );
-                  }
+                  (fingerprint: string, index: number) => (
+                    <AssetTokenInput
```
Comment (nikolaglumac) [inline source/renderer/app/components/wallet/WalletSendForm.js:804]: @yakovkaravelov please rename this component to `AssetInput`

File: source/renderer/app/components/wallet/WalletSendForm.js [hunk 2]
```diff
@@ -804,167 +800,27 @@ export default class WalletSendForm extends Component<Props, State> {
...
+                      key={fingerprint}
+                      fingerprint={fingerprint}
+                      index={index}
+                      getAssetByFingerprint={this.getAssetByFingerprint}
+                      availableAssets={this.availableAssets}
+                      receiverFormField={formFields.receiver}
```
Comment (nikolaglumac) [inline source/renderer/app/components/wallet/WalletSendForm.js:810]: @yakovkaravelov we should not pass the whole `receiverFormField`. It should be enough to pass just the individual `assetField` and the `assetsDropdown`.

**Review by nikolaglumac — CHANGES_REQUESTED** [review]
@yakovkaravelov can we apply the same dynamic width calculation for the ada amount field too? 🙏 

Screenshot 2021-03-02 at 18 46 52

**Comment by nikolaglumac** [comment]
@darko-mijic let's also update this tooltip text copy so that it doesn't mention other assets in case user is sending only ada.

Screenshot 2021-03-02 at 18 48 00

**Comment by nikolaglumac** [comment]
@darko-mijic can you please help us with https://github.com/input-output-hk/daedalus/pull/2422#issuecomment-789090959

**Comment by yakovkaravelov** [comment]
This is finished. Please review, @gabriela-ponce @tomislavhoracek Thanks.

**Review by nikolaglumac — APPROVED** [review]
Great work 🎉

**Review by gabriela-ponce — APPROVED** [review]
Looks good 🎉

**Pull request merged**